transifex/native/cds.py

```diff
@@ -108,10 +122,15 @@ def fetch_translations(self, language_code=None):
 
             for language_code in set(languages) & \
                     set(self.configured_language_codes):
+
+                req_headers = {"Authorization": "Bearer {token}".format(
+                        token=self.token),
+                        "If-None-Match": self.etags.get(language_code, '')
```

Comment:
`self.etags.get` accepts one argument. It already returns `''` as a fallback value so simply lose the second argument.

Reply:
@kbairak that is true ...